packages/components/src/public/QueryModel/grid/utils.ts

```diff
@@ -4,9 +4,20 @@ import { Filter } from '@labkey/api';
 import { QueryColumn } from '../../..';
 
 import { ActionValue } from './actions/Action';
-import { Change, ChangeType } from './OmniBox';
 import { SearchAction } from './actions/Search';
 
+export enum ChangeType {
+    add = 'add',
+    remove = 'remove',
+    modify = 'modify',
+    none = 'none',
+}
+
+export interface Change {
+    type: ChangeType;
+    index?: number;
+}
```

Comment:
I wouldn't necessarily expect to find interfaces and enums in a `utils` file, but no biggie.

Reply:
good call. moved to a new `model.ts` file